### ISSUE TITLE
how to use it with nginx

### ISSUE THREAD

**Issue by drduan** [selfcertificationhub/selfcertificationhub#1]

**Comment by selfcertificationhub** [comment]
I wrote a document. Can you take a look at it? Here is the [link: Applying a Self-Signed HTTPS Certificate to Nginx
](https://github.com/selfcertificationhub/selfcertificationhub/blob/main/docs/Q%26A/Applying_a_Self-Signed_HTTPS_Certificate_to_Nginx.md).

**Comment by drduan** [comment]
![image](https://github.com/selfcertificationhub/selfcertificationhub/assets/7875572/7c7aa696-f7e8-4746-91de-03c30155d130)


remark:::  I think u should add a export function ,and save ```Certification ```   and ```Certification key ```   to server.crt or server.key

**Comment by selfcertificationhub** [comment]
Yes, you’re right. It’s in my [roadmap](https://github.com/selfcertificationhub/selfcertificationhub/blob/main/README.md).
 I will find time to add this feature.
